src/docs/release-notes/R18/R18.mdx

```diff
@@ -1,10 +1,36 @@
+### Component Libraries
+
+-   Added a [Mobile Stepper](https://pxblue-components.github.io/angular/?path=/info/components-mobile-stepper--readme) component for Angular
+-   Angular components are now compatible with Angular 10+
+-   Updated React Native components to be compatible with react-native-paper version 4
+
+### Design Patterns
+
+-   Updated content for [Navigation](/patterns/navigation) and [Visualizations](/patterns/visualizations)
+-   Released a new [Design Pattern Stickersheet](https://www.figma.com/community/file/926189711301522231) that will include many examples of all of our [Design Patterns](/patterns), starting with [forms](/patterns/forms), [side sheet](/patterns/overlay) and [lists](/patterns/lists).
+
+### Documentation Site≈
+
+-   Updated [Iconography](/style/iconography) page to enable users to download icons and search by keywords/tags
+-   Added a new page dedicated to helping you build a [Project Identity](/design/project-identity)
+-   Added new holiday themes
+
+### PX Blue CLI
+
+-   You can now use [templates](https://github.com/pxblue/react-cli-templates) to kick-start your react projects with navigation and authentication
+
+### Reusable Workflows
+
+-   Reusable [Authentication Workflow](https://www.npmjs.com/package/@pxblue/angular-auth-workflow) for Angular.
+
 ### Stickersheet Components
```

Comment:
We outa find some more consistent names for those Figma files

DSM: (Figma) Component Stickersheet
The new design pattern file: (Figma) Design Pattern Stickersheet

Reply:
And end the fight of `stickersheet` vs `sticker sheet`